staramr/blast/BlastHandler.py

```diff
@@ -119,6 +128,24 @@ def _make_db_from_input_files(self, db_dir, files):
 
         return db_files
 
+    def _schedule_mlst(self, path, file: list) -> None:
+
+        curr_files = copy.deepcopy(file);
+        num_threads_param = ("{}").format(self._threads);
+
+        command = ['mlst', '--threads']
```

Comment:
This is for later if we have time to revisit, but I wanted to make a note of it. I don't think `--threads` for the `mlst` program works as I expected and there's likely some other improvements we could make to have this run faster. But, this is for a different time since it all works right now and is still pretty fast.

Reply:
I added an issue for this #81